lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -301,11 +300,11 @@
 
         &:hover,
         &:focus {
-          background: $gray-l5 !important;
+          border-bottom: 3px solid $active-color;
```

Comment:
Nit: it would be great to get rid of `$active-color` as it should just be `theme-color("primary")`.